### DISCUSSION TITLE
Plex Servers: Implement the "Unified View"

### DISCUSSION THREAD

**Discussion by wes1993** [pannal/plex-for-kodi#103] [Ideas]
Is possible to Unify all plex servers in one view? like the stock app do

Instead of switch every time from one server to another have just one views with all servers inside.

Best Regards
Stefano

**Comment by pannal** [comment]
It should be possible but would require a pretty massive rewrite. It's not on my current list of things to do because of its size. Maybe someone else will tackle it.

**Comment by wes1993** [comment]
I hope, if someone need I can help a little with python programming :-D
